### ISSUE TITLE
Missing special case calc handling for -ms-calc()

### ISSUE THREAD

**Issue by xtianus79** [sass/libsass#842]
Special case `calc` handling needs to be implemented for the `-ms-calc()` function to prevent `Incompatible units` errors.

```scss
foo {
  foo: -ms-calc(100% - 100px);
}
```

expected

```css
foo {
  foo: -ms-calc(100% - 100px); }
```

actual

```css
Incompatible units: 'px' and '%'
```

**edited by @xzyfer for clarification**

**Comment by xzyfer** [comment]
Can you please create a http://sassmeister.com/ the shows the issue?

**Comment by umyeah** [comment]
I ran into this issue as well when trying to do width: -ms-calc(100% - 100px); 

This is happening to me when using libsass 3.1.0 on sassmeister as well as when using node-sass version 2.0.1 (not sure what version of libsass that is using)

**Comment by xzyfer** [comment]
Thanks for the clarification @umyeah. I can confirm that this issue is still present on master.

**Comment by am11** [comment]
@umyeah, you can check the version of node-sass and corresponding version of libsass as follow:

CLI usage:

```cmd
node-sass --help
```

Code usage:

```javascript
require('node-sass').info()
```

node-sass v2.0.1 corresponds to libsass v3.1.0.

**Comment by xzyfer** [comment]
I've confirmed this is still an issue on master.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/264

**Comment by xzyfer** [comment]
This is fixed and will be in 3.2
